services/fileuploader/fileuploader.go

```diff
@@ -98,6 +102,11 @@ func (p *provider) updateLoop(ctx context.Context, backendConfig backendconfig.B
 				bucket = overrideWithSettings(defaultBucket.Config, settings, workspaceId)
 			} else {
 				bucket = getDefaultBucket(ctx, config.GetString("JOBS_BACKUP_STORAGE_PROVIDER", "S3"))
+				switch c.Settings.DataRetention.RetentionPeriod {
+				case "default":
+					bucket.Config["prefix"] = config.GetString("JOBS_BACKUP_STORAGE_DEFAULT_PREFIX", "7dayretention")
+				default:
```

Comment:
Should we add the other case for completeness 
```suggestion
				case "default":
					bucket.Config["prefix"] = config.GetString("JOBS_BACKUP_STORAGE_DEFAULT_PREFIX", "7dayretention")
				case "full":
				default:
```